website/config.rb

```diff
@@ -19,7 +19,7 @@
 helpers do
   # get correct analytics id
   def segmentId()
-    if (ENV['ENV'] == 'production')
+    if (ENV['ENV'] == 'tmp-production')
       'OdSFDq9PfujQpmkZf03dFpcUlywme4sC'
     else
       '0EXTgkNx0Ydje2PGXVbRhpKKoe5wtzcE'
```

Comment:
Are these safe to be in the public repo? Not sure what service is consuming them. Would it be possible for anyone to use them to maliciously generate and send fake data?

Reply:
Very reasonable question. These Segment IDs are added to the tracking script injected into the page and are used client side, so are exposed intentionally.

Reply:
Perfect!